@djdebonis output of `make lint`, formatted as a list.

- [x] `src/cafe/farm.py:5:1: F401 'random' imported but unused`
- [x] `src/cafe/farm.py:6:1: F401 'statistics as stats' imported but unused`
- [x] `src/cafe/farm.py:7:1: F401 'typing.Dict' imported but unused`
- [x] `src/cafe/farm.py:40:101: E501 line too long (108 > 100 characters)`
- [x] `src/cafe/simulateCoOp.py:1:1: F401 'pandas as pd' imported but unused`
- [x] `src/cafe/simulateCoOp.py:2:1: F401 'statistics as stats' imported but unused`
- [x] `src/cafe/simulateCoOp.py:6:1: F401 'cafe.importData' imported but unused`
- [x] `src/cafe/simulateCoOp.py:19:5: F841 local variable 'numPlots' is assigned to but never used`
- [x] `src/cafe/simulateCoOp.py:60:5: F841 local variable 'trees' is assigned to but never used`
- [x] `src/cafe/simulateCoOp.py:61:5: F841 local variable 'strategy' is assigned to but never used`
- [x] `src/cafe/simulateCoOp.py:69:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:72:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:74:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:76:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:122:101: E501 line too long (115 > 100 characters)`
- [x] `src/cafe/simulateCoOp.py:123:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:125:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:135:101: E501 line too long (106 > 100 characters)`
- [x] `src/cafe/simulateCoOp.py:136:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:138:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:147:101: E501 line too long (124 > 100 characters)`
- [x] `src/cafe/simulateCoOp.py:148:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:150:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:160:101: E501 line too long (103 > 100 characters)`
- [x] `src/cafe/simulateCoOp.py:161:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:163:1: W293 blank line contains whitespace`
- [x] `src/cafe/simulateCoOp.py:174:1: W293 blank line contains whitespace`
- [x] `src/cafe/importData.py:3:1: F401 'cafe.farm' imported but unused`
- [x] `src/cafe/importData.py:8:101: E501 line too long (133 > 100 characters)`
- [x] `src/cafe/fakeData.py:11:101: E501 line too long (105 > 100 characters)`
- [x] `src/cafe/fakeData.py:119:1: W293 blank line contains whitespace`
- [x] `src/cafe/fakeData.py:129:1: W293 blank line contains whitespace`
- [x] `src/cafe/fakeData.py:130:101: E501 line too long (115 > 100 characters)`
- [x] `src/cafe/fakeData.py:132:1: W293 blank line contains whitespace`
- [x] `src/cafe/fakeData.py:141:101: E501 line too long (106 > 100 characters)`
- [x] `src/cafe/fakeData.py:142:1: W293 blank line contains whitespace`

--- a/makefile
+++ b/makefile
@@ -20,9 +20,9 @@ cov:
 
 clean:
 	rm -rf tests/.ipynb_checkpoints
-	rm .coverage
-	rm .pytest_cache/
-	rm -rf .eggs
+	rm -f  .coverage
+	rm -rf .pytest_cache/
+	rm -rf .eggs/
 	rm -rf .ipynb_checkpoints
 	rm -rf src/cafe/.ipynb_checkpoints
 	rm -rf binder/.ipynb_checkpoints

--- a/src/cafe/fakeData.py
+++ b/src/cafe/fakeData.py
@@ -5,12 +5,16 @@ from datetime import date
 
 """
 
-Copmpile a fake dataset resembling the dataset Michael collected on his trip to Guatemala in.
+Copmpile a fake dataset resembling the dataset
+Michael collected on his trip to Guatemala in.
 
-The fake dataset uses relative proportions of the tree types in the original dataset as
-probabilities for selection of the tree types of this dataset. So, although it is not the original data--
+The fake dataset uses relative proportions of the
+tree types in the original dataset as
+
+probabilities for selection of the tree types of this dataset.
+
+So, although it is not the original data--
 and the names are fake!--it resembles the patterns of the co-op.
-
 """
 # get current year for auto assgn
 currentDate = date.today()
@@ -105,8 +109,9 @@ if __name__ == "__main__":
         default=100,
         type=int,  # string type works well for
         help="""
-                        Number of farms (i.e. the number of rows) desired for the output dataset.
-                        """,
+            Number of farms (i.e. the number of rows)
+            desired for the output dataset.
+        """,
     )
 
     parser.add_argument(
@@ -115,9 +120,8 @@ if __name__ == "__main__":
         default=40,
         type=int,  # string type works well for
         help="""
-                        Number of fake names (farmers) desired as owners of of the farms.
-                        
-                        """,
+            Number of fake names (farmers) desired as owners of of the farms.
+        """,
     )
 
     parser.add_argument(
@@ -126,11 +130,10 @@ if __name__ == "__main__":
         default=defaultYear,
         type=int,  # string type works well for
         help="""
-                        
-                        The year in which the simulation will begin (this is important because all of the trees are
-                        'planted' relative to this date)
-                        
-                        """,
+            The year in which the simulation will begin
+            (this is important because all of the trees are
+            'planted' relative to this date)
+        """,
     )
     parser.add_argument(
         "-o",
@@ -138,9 +141,9 @@ if __name__ == "__main__":
         default="data/fakeData.csv",
         type=str,  # string type works well for
         help="""
-                        Desired name of file (and path from current directory) for the output spreadsheet.
-                        
-                        """,
+            Desired name of file (and path from current directory)
+            for the output CSV file.
+        """,
     )
 
     args = parser.parse_args()

--- a/src/cafe/farm.py
+++ b/src/cafe/farm.py
@@ -2,9 +2,7 @@ from dataclasses import dataclass, field
 import datetime
 from functools import lru_cache
 import math
-import random
-import statistics as stats
-from typing import List, Dict, Tuple
+from typing import List, Tuple
 from numbers import Number
 import warnings
 
@@ -37,7 +35,11 @@ def find_config(name: str, configs: Tuple[Config]) -> Config:
     # if none found, seek species default
     # TODO print warning about this behavior
     warnings.warn(
-        f"Could not find canonical match for species=`{name}`, searching for match against species instead."
+        (
+            "Could not find canonical match"
+            f"for species=`{name}`, searching for"
+            "match against species instead."
+        )
     )
     for c in configs:
         if c.species == name:

--- a/src/cafe/importData.py
+++ b/src/cafe/importData.py
@@ -1,12 +1,11 @@
 import pandas as pd
 import yaml
-import cafe.farm as farm
 
 
 def readData(filePath: str):
     """
-    Takes a filepath in the form of a string (e.g. 'data/demoData.csv') that represents a spreadsheet and returns a pandas dataframe.
-
+    Takes a filepath in the form of a string (e.g. 'data/demoData.csv')
+    that represents a spreadsheet and returns a pandas dataframe.
     """
     pathExt = filePath.split(".")
     ext = pathExt[1]

--- a/src/cafe/simulateCoOp.py
+++ b/src/cafe/simulateCoOp.py
@@ -1,9 +1,6 @@
-import pandas as pd
-import statistics as stats
 import matplotlib.pyplot as plt
 
 from cafe.farm import Config, Farm, predict_yield_for_farm
-import cafe.importData as importData
 
 
 def simulateCoOp(plotList, numYears, pruneYear=None, growthPattern=None, strategy=None):
@@ -16,7 +13,7 @@ def simulateCoOp(plotList, numYears, pruneYear=None, growthPattern=None, strateg
 
     """
 
-    numPlots = len(plotList)
+    # numPlots = len(plotList)
 
     annualHarvest = []
     harvestYear = []
@@ -57,25 +54,22 @@ def main(args):
     import os
 
     farmData = args.farm
-    trees = args.trees
-    strategy = args.strategy
+    # trees = args.trees
+    # strategy = args.strategy
     years = args.years
     output = args.output
 
     if not os.path.exists(farmData):
         raise ValueError(
-            """
-        File: %s does not exist
-        
-        If you are running default commands and this is your first time
-        running the simulation, assure you have run:
-        
-        `python3 src/cafe/fakeData.py --farms 100 --year 2020 --output data/fakeData.csv`
-        
-        in the core directory before calling simulateCoOp.py from the command line.
-        
-        """
-            % farmData
+            (
+                f"File: {farmData} does not exist.\n"
+                "If you are running default commands and this is your first time"
+                "running the simulation, assure you have run:\n"
+                "`python3 src/cafe/fakeData.py --farms 100"
+                "--year 2020 --output data/fakeData.csv`\n"
+                "in the core directory before calling"
+                "simulateCoOp.py from the command line."
+            )
         )
 
     print("Importing Data")
@@ -117,62 +111,53 @@ if __name__ == "__main__":
         "-f",
         "--farm",
         default="data/fakeData.csv",
-        type=str,  # string type works well for
+        type=str,
         help="""
-                        Name of file (and path from current directory) to data containing cuerdas, tree types, etc.
-                        
-                        Example (& default): --farm data/demoData.csv
-                        
-                        """,
+           Path to data containing plot details.
+           e.g.,  cuerdas, tree types, etc.\n
+        """,
     )
 
     parser.add_argument(
         "-t",
-        "--trees",  # currently this information is stored in the Cuerdas class
+        "--trees",
         default="data/trees.yml",
-        type=str,  # string type works well for
+        type=str,
         help="""
-                        Name of file (and path from current directory) to data containing tree attributes.
-                        
-                        Example (& default): --trees data/trees.yml
-                        
-                        """,
+            Name of file (and path from current directory)
+            to data containing tree attributes.\n
+            Example (& default): --trees data/trees.yml
+        """,
     )
     parser.add_argument(
         "-s",
         "--strategy",
         default="data/strategy1.yml",
-        type=str,  # string type works well for
+        type=str,
         help="""
-                        Name of file (and path from current directory) to data containing method, strategy, & approach data.
-                        
-                        Example (& default): --strategy intervention/strategy1.yml
-                        
-                        """,
+            Name of file (and path from current directory)
+            to data containing method, strategy, & approach data.\n
+            Example (& default): --strategy intervention/strategy1.yml
+        """,
     )
 
     parser.add_argument(
         "-y",
         "--years",
         default=30,
-        type=int,  # string type works well for
+        type=int,
         help="""
-                        Number of years that should be iterated through in the simulation (type : int).
-                        
-                        Example (& default): --year 30
-                        
-                        """,
+            Number of years that should be iterated
+            through in the simulation (default=30).\n
+        """,
     )
 
     parser.add_argument(
         "-o",
         "--output",
         default="testNewFarm.png",
-        type=str,  # string type works well for
-        help="""
-                        Desired name of plot output file.
-                        
-                        """,
+        type=str,
+        help="Desired name of plot output file (default=testNewFarm.png).",
     )
 
     args = parser.parse_args()

--- a/tests/test_fakeData.py
+++ b/tests/test_fakeData.py
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-import pytest
+# import pytest
 import cafe.fakeData as fd
 from unittest import mock
 import pandas as pd

--- a/tests/test_farm.py
+++ b/tests/test_farm.py
@@ -11,7 +11,7 @@ from cafe.farm import (
 )
 
 # import cafe.importData as importData
-import warnings
+# import warnings
 
 from unittest import mock
 from pandas import DataFrame
@@ -74,6 +74,7 @@ def test_that_membership_is_based_on_species_and_name_when_both_supplied():
 def test_error_thrown_when_bad_config_requested(configs):
     with pytest.raises(ValueError):
         c = find_config("f", configs)
+    assert c is None
 
 
 @pytest.mark.filterwarnings("ignore::UserWarning")

